elfpy/utils/fmt.py

```diff
@@ -0,0 +1,35 @@
+"""
+This module contains a function to format a float to a string with a given precision
+"""
+import numpy as np
+
+
+def fmt(value, precision=3, min_digits=0, debug=False):
```

Comment:
I would rename this to `float_to_string` and if you want a shorter import than import as `from fmt import format_to_string as fmt`

Reply:
actually rename the file too, `fmt.py` not very descriptive.

Reply:
I think it makes more sense to put this function into ouputs.py

Reply:
done in 2d73538bc0258631b6b16373cf8b217d2ee42198